### DISCUSSION TITLE
windows install

### DISCUSSION THREAD

**Discussion by alceausu** [skypilot-org/skypilot#1994] [Q&A] [answered]
Does it work on Windows?
It works configuring the cloud, then sky check just fine, but 
I have a error on windows running the 'hello_sky.yaml' tutorial
`
"    raise FileNotFoundError(
FileNotFoundError: Could not find built SkyPilot wheels with glob pattern **/skypilot-0.2.5-*.whl under WindowsPath(..."
`
I tried to install from source, not working either: 
`
NameError: name 'sky' is not defined
`

**Comment by romilbhardwaj** [comment, marked as answer]
Hi @alceausu - we're [investigating](https://github.com/skypilot-org/skypilot/pull/1955#issuecomment-1553783525) supporting windows. While we work on it, is it possible to use WSL? For Windows users, our recommended and tested way of using SkyPilot is through WSL.

**Reply by alceausu** [reply]
It works with WSL. Thank you.

**Comment by yuval-airis** [comment]
I got into the same issue, and solved it by moving to WSL
